libs/ledger-live-common/src/wallet-api/converters.ts

```diff
@@ -33,6 +33,9 @@ export function accountToWalletAPIAccount(
 
     return {
       id: walletApiId,
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore Remove once wallet-api
+      parentAccountId: uuidv5(parentAccount.id, NAMESPACE),
```

Comment:
Actually you might want to move the `uuidv5` above and also do a `uuidToAccountId.set(WalletApiParentId, parentAccount.id);` before return the account